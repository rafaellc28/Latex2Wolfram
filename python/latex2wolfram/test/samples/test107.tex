% Test cross expression
\begin{pmatrix}
  \frac{1}{4}\\
  -\frac{1}{2}\\
  1
\end{pmatrix} \times
\begin{pmatrix}
  \frac{1}{2}\\
  1\\
  -\frac{2}{3}\\
\end{pmatrix}
