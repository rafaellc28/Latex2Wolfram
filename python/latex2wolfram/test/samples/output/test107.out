{{1/4}, {-1/2}, {1}} x {{1/2}, {1}, {-2/3}}
